### ISSUE TITLE
Is the noise addition repeated？

### ISSUE THREAD

**Issue by Love-Sunshine** [prabhatkc/ct-recon#2]
Thank you very much for the code provided, but I have identified several minor issues.
In file insert_noise_2_obj_model.m, have both pct_ldsino and yuct_ldsino added noise repeatedly? Why is k_nd determined to be 0.85? The ndsino_n in im(3, ndsino_n, 'sino'), cbar; also does not appear in the code. How can these problems be solved?
Thank you very much for your answer!

**Comment by prabhatkc** [comment]
Firstly, I have fixed the plotting issues in the file, insert_noise_2_obj_model.m
and updated GitHub with the corresponding fixes. 
Secondly, yes, both the functions, pct_ldsino and yuct_ldsino, add noise. pct_ldsino yields normal-dose acquisition (which still contains some level of noise), and then yuct_ldsino further adds noise to acquisition from normal-dose to yield low-dose measurement. Refer to the references cited in the header part in the same Matlab file. 
Next, k_nd=0.85 was heuristically determined to match the NPS curve obtained from the uniform scan provided in the low-dose grand challenge dataset. Look at the part https://github.com/prabhatkc/ct-recon/tree/main/error_analysis/cho_lcd#nps-validation-against-ldgc to get more idea on this aspect. 
Finally, I have uploaded a demo file, https://github.com/prabhatkc/ct-recon/blob/main/digiNoise/demo_m1_validation.m, with a noiseless reconstruction example.  
I hope this info helps.
